src/gui/properties/propertieswidget.cpp

```diff
@@ -511,77 +516,77 @@ void PropertiesWidget::loadUrlSeeds()
 {
     m_ui->listWebSeeds->clear();
     qDebug("Loading URL seeds");
-    const QList<QUrl> hc_seeds = m_torrent->urlSeeds();
+    const QList<QUrl> hcSeeds = m_torrent->urlSeeds();
     // Add url seeds
-    foreach (const QUrl &hc_seed, hc_seeds) {
-        qDebug("Loading URL seed: %s", qUtf8Printable(hc_seed.toString()));
-        new QListWidgetItem(hc_seed.toString(), m_ui->listWebSeeds);
+    foreach (const QUrl &hcSeed, hcSeeds) {
+        qDebug("Loading URL seed: %s", qUtf8Printable(hcSeed.toString()));
+        new QListWidgetItem(hcSeed.toString(), m_ui->listWebSeeds);
     }
 }
 
 void PropertiesWidget::openDoubleClickedFile(const QModelIndex &index)
 {
-    if (!index.isValid()) return;
-    if (!m_torrent || !m_torrent->hasMetadata()) return;
-    if (PropListModel->itemType(index) == TorrentContentModelItem::FileType)
+    if (!index.isValid() || !m_torrent || !m_torrent->hasMetadata()) return;
```

Comment:
what is the point? if none, please refrain from doing so in the future.

Reply:
ok...